src/Lumper.UI/Views/Tasks/TasksView.axaml

```diff
@@ -0,0 +1,59 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:converters="clr-namespace:Lumper.UI.Converters"
+             xmlns:taskviews="clr-namespace:Lumper.UI.Views.Tasks"
+             xmlns:tasks="clr-namespace:Lumper.Lib.Tasks"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:Class="Lumper.UI.Views.Tasks.TasksView">
+    <UserControl.Resources>
+        <converters:TaskStatusConverter x:Key="taskStatus" />
+        <converters:PercentConverter x:Key="percent" />
+    </UserControl.Resources>
+    <Grid RowDefinitions="Auto, *, Auto" ColumnDefinitions="*,2*">
+        <DockPanel Grid.Row="0" Grid.Column="0">
+            <Button DockPanel.Dock="Left" Click="OnLoadClick" IsEnabled="{Binding !IsRunning}">Load</Button>
+            <Button DockPanel.Dock="Left" Click="OnSaveClick" IsEnabled="{Binding !IsRunning}">Save</Button>
```

Comment:
Let's make it clearer what this is doing. We have horizontal space free, let's do "Load Tasks"/"Save Tasks"